biogtr/training/configs/README.md

```diff
@@ -0,0 +1,180 @@
+We force the tracker to assign instances to a track instead of creating a new track if `max_tracks `has been reached.
+
+## `runner`
+
+This section contains parameters for how to handle training/validation/testing
+
+### `metrics`
+
+This section contains config for which metrics to compute during training/validation/testing. See [`pymotmetrics.list_metrics`](https://github.com/cheind/py-motmetrics) for available metrics.
+
+Should have a `train`, `val` and `test` key with corresponding list of metrics to compute during training.
+
+### `persistent_tracking`
+
+This section indicates whether or not to track across chunks during training/validation/testing
+
+Should have a `train`, `val` and `test` key with a corresponding `bool` indicating whether to use persistent tracking.
+
+### `dataset`
+
+This section contains the params for initializing the datasets for training. Requires a `train_dataset` and optionally `val_dataset`, `test_dataset` keys. For the correct args corresponding to the labels and vid_files paths as well as specific `kwargs` please see the corresponding `biogtr.datasets` submodule. For instance `SleapDataset` requires a `slp_files` and `video_files` key while `MicroscopyDataset` requires `videos` and `tracks`. Below we describe the common parameters regardless of dataset
+
+* `padding`: An `int` representing the amount of padding to be added to each side of the bounding box size
+* `crop_size`: Either an `int` or `tuple` indicating the size of the bounding box around which a crop will form.
+* `chunk`: Whether or not to chunk videos into smaller clips to feed to model
+* `clip_length`: the number of frames in each chunk
+* `mode`: `train` or `val`. Determines whether this dataset is used for training or validation.
+* `n_chunks`: Number of chunks to subsample from. Can either a fraction of the dataset (ie `(0,1.0]`) or number of chunks
+* `seed`: set a seed for reproducibility
+* `gt_list`: An optional path to .txt file containing ground truth for cell tracking challenge datasets.
+
+#### `dir`:
+This section allows you to pass a directory rather than paths to labels/videos individually
+
+* `path`: The path to the dir where the data is stored (recommend absolute path)
+* `labels_suffix`: A `str` containing the file extension to search for labels files. e.g. `.slp`, `.csv`, or `.xml`.
+* `vid_suffix`: A `str` containing the file extension to search for video files e.g `.mp4`, `.avi` or `.tif`.
+#### `augmentations`:
+
+This subsection contains params for albumentations. See [`albumentations`](https://albumentations.ai) for available visual augmentations. Other available augmentations include `NodeDropout` and `InstanceDropout`. Keys must match augmentation class name exactly and contain subsections with parameters for the augmentation e.g.
+
+```YAML
+augmentations: 
+    Rotate:
+        limit: 45
+        p: 0.3
+    ...
+    MotionBlur:
+        blur_limit: [3,7]
+        p: 0.3
+```
```

Comment:
This is great! A concrete example for how this subsection should be written. Have more of these everywhere.